modules/models/behavior/constant_velocity/constant_velocity.hpp

```diff
@@ -34,7 +34,7 @@ class BehaviorConstantVelocity : public BehaviorIDMClassic {
   Trajectory Plan(float delta_time,
                   const ObservedWorld& observed_world);
 
-  std::pair<double, double> GetTotalAcc(
+  virtual std::pair<double, double> GetTotalAcc(
```

Comment:
does this need to be virtual?
https://en.cppreference.com/w/cpp/language/virtual

Otherwise should inherit from BaseIDM.